docs/motivation.md

```diff
@@ -10,7 +10,7 @@ Basically, there are a few types of forms with different capabilities.
 Most of the time you’ll be using either AutoForm or ValidatedForm, but there are quite a few more to choose from:
 
 <p align="center">
-  <img src="/img/uniforms-graph.svg" />
+  <img src="/img/uniforms-graph.svg" alt="Graph showing inheritance hierarchy of different forms." />
```

Comment:
I don't think these alts are good. You should try imagining in a situation where you just cannot see this image at all and see if the alt gives you any useful information.

This as an example: Above this image you have a mention "There’s a very interesting class-based inheritance concept for forms." so we already know there is inheritance in uniforms.

Now we have an image with an alt that says "Graph showing inheritance hierarchy of different forms.". We already know there is an inheritance concept from the text above. Does this alt add any additional useful info? We get to know that a graph is supposed to be here, and it's supposed to be showing forms. But how does the graph look like or what kind of forms are present on it will never be known to the person reading this alt. This is why in my opinion this description is not useful and either should be rephrased to specify exactly what is contained within the graph or the alt should be empty.

Reply:
I consulted this with content creator and the new proposition is: "AutoForm class inheritance from ValidatedQuickForm which subsequently inherits from QuickForm and ValidatedForm"